ui/client/components/Process/ProcessTypes.ts

```diff
@@ -18,9 +17,10 @@ export type ProcessAction = {
   buildInfo?: {};
 }
 
-export interface Process {
-  id: number;
+export interface ProcessType {
```

Comment:
why do we need Type in all these places?

Reply:
For example we have component like Process - it is misleading.. We have more that kind examples.. I think we should create some standard for it.. *Type defines in a good way funcionality these elements.